Things.md

```diff
@@ -0,0 +1,22 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are manipulated and explain how the GroupHoldr system is used.  
+  
+###Table of Contents
+1. [Manipulating Things](#manipulating-things)
+	1. [Shifting Things](#shifting-things)
+	2. [Using the GroupHoldr System](#using-the-groupholr-system)
+
+##Manipulating Things
+Everything you see in the game (trees, blocks, the player, etc.) is a Thing. The Thing class is subclassed by a new class for everything (Tree class, Block class, Player class, etc.). These Things can be manipulated to change their properties, placements, etc.  
+###Shifting Things  
```

Comment:
Copy the pre-existing .md files' conventions for endlines around paragraphs and headers